components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -67,10 +67,15 @@ void RRTMGPRadiation::set_grids(const std::shared_ptr<const GridsManager> grids_
   add_field<Required>("p_mid" , scalar3d_layout_mid, Pa, grid->name(), ps);
   add_field<Required>("p_int", scalar3d_layout_int, Pa, grid->name(), ps);
   add_field<Required>("pseudo_density", scalar3d_layout_mid, Pa, grid->name(), ps);
-  add_field<Required>("sfc_alb_dir_vis", scalar2d_layout, nondim, grid->name());
-  add_field<Required>("sfc_alb_dir_nir", scalar2d_layout, nondim, grid->name());
-  add_field<Required>("sfc_alb_dif_vis", scalar2d_layout, nondim, grid->name());
-  add_field<Required>("sfc_alb_dif_nir", scalar2d_layout, nondim, grid->name());
+  // WARNING! We are switch surface albedo variables to "Updated" to allow rrtmgp
```

Comment:
Imho, we should just add FPC's in the SC class (it's not complicated). This is a hack that I'm afraid might be forgotten, and left there...

Reply:
I'll let @PeterCaldwell chime in.  We decided this route to make for the quickest implementation so that Noel could start using it right away.  It would also be simple to undo, hence the WARNING in the code.  My hope is that this comment is direct enough that we won't see it and move on (like so much EAM code), and that we will revert when we make SFC CPL an atmosphere process.  I would like to think that this will be on our minds when we do eventually get the SFC CPL as ATM PROC PR in.

Reply:
We should keep an issue or a confluence page, tracking all the "temporary hacks" we put in the code for the sake of getting stuff to run. I am always wary of "we'll fix this later" promises. I've been making that kind of promise myself in the past... ;)